File: server/router/admin/gost_client.go
package admin

import (
	"github.com/gin-gonic/gin"
	"server/controller/admin/gost_client"
	"server/global"
	"server/router/middleware"
)

func InitGostClient(group *gin.RouterGroup) {
	g := group.Group("gost/client", middleware.Auth(global.Jwt), middleware.AuthAdmin())
	g.POST("create", gost_client.Create)
	g.POST("delete", gost_client.Delete)
	g.POST("page", gost_client.Page)
	g.POST("list", gost_client.List)
}

File: web/src/api/admin/gost_client.js
import request from "../../request/index.js";

const baseUrl = '/v1/admin/gost/client'

export const apiAdminGostClientPage = (data) => {
    return request.request({
        url: `${baseUrl}/page`,
        method: 'POST',
        data
    })
}

export const apiAdminGostClientList = (data) => {
    return request.request({
        url: `${baseUrl}/list`,
        method: 'POST',
        data
    })
}

export const apiAdminGostClientCreate = (data) => {
    return request.request({
        url: `${baseUrl}/create`,
        method: 'POST',
        data
    })
}

export const apiAdminGostClientDelete = (data) => {
    return request.request({
        url: `${baseUrl}/delete`,
        method: 'POST',
        data
    })
}